src/tag.c

```diff
@@ -3705,6 +3708,9 @@ jumpto_tag(
     size_t	len;
     char_u	*lbuf;
 
+    if (postponed_split == 0 && !check_can_set_curbuf_forceit(forceit))
+        return FAIL;
```

Comment:
and here please

Reply:
We may be already to remove this block since we now have a check inside of `getfile` but I think I would need add a test to ensure splits are still happening for tag commands. We do this already in certain other places like the `cnext` and `xdo` unittests but I don't believe I added any "did it successfully split" unittests for tag-related commands. Please let me know if you'd like those

Reply:
okay, if you could please add that additional test here then?

Reply:
Sure thing. Looking at the current unittests, this actually only seemed to apply to `Test_normal_ctrl_w_ctrl_square_bracket_right` and `Test_normal_ctrl_w_g_ctrl_square_bracket_right`. I've added to both.